tests/test_elasticsearch/elasticsearch_test.sh

```diff
@@ -37,22 +43,34 @@ docker run -d --rm --name=tfio-elasticsearch \
 --ulimit memlock=-1:-1 \
 ${ELASTICSEARCH_IMAGE}
 
+echo ""
 echo "Waiting for the elasticsearch cluster to be up and running..."
+echo ""
 sleep 20
 
+echo ""
 echo "Checking the base REST-API endpoint..."
-curl localhost:9200/
+echo ""
+curl -X GET localhost:9200/ --header 'Authorization: Basic ZWxhc3RpYzpkZWZhdWx0X3Bhc3N3b3Jk'
```

Comment:
Where did `ZWxhc3RpYzpkZWZhdWx0X3Bhc3N3b3Jk` come from?

Reply:
addressed in the commit.

Reply:
I am setting the password for the `elastic` user using env vars. So it is the base64 encoded string of "elastic:default_password". Used only for testing purposes.